lib/hydra/works/indexers/collection_indexer.rb

```diff
@@ -0,0 +1,9 @@
+module Hydra::Works
+  class CollectionIndexer < Hydra::PCDM::CollectionIndexer
```

Comment:
Unfortunately I don't think we can do that, because of the inheritance. Wouldn't GenericWorks need the GenericObjectIndexer from PCDM?

We probably want to rearchitect the indexers to avoid duplication via composition, but I'd have to start that lower down the stack.